tasks/section3.yml

```diff
@@ -48,6 +50,8 @@
   with_items:
       - { name: net.ipv4.conf.all.accept_source_route, value: 0 }
       - { name: net.ipv4.conf.default.accept_source_route, value: 0 }
+      - { name: net.ipv6.conf.all.accept_source_route, value: 0 }
```

Comment:
better separate ipv4 and ipv6 into two tasks in one block

Reply:
Separated